test/Mono.Linker.Tests.Cases.Expectations/Assertions/ExpectGeneratedStringAttribute.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+
+namespace Mono.Linker.Tests.Cases.Expectations.Assertions
+{
+	public class ExpectGeneratedStringAttribute : BaseMemberAssertionAttribute
```

Comment:
I think these need comments describing what they do (and in which tests they actually work).
And I also think the names should mention the "Documentation syntax" in some way - when I saw this I could not figure out what the attribute is for until I saw it being used.